### PR TITLE
doc: add minutes for meeting 24 Sep 2020

### PR DESCRIPTION
**Pull request by mhdawson**
Signed-off-by: Michael Dawson <mdawson@devrus.com>

### CODE REVIEW

**Comment by mhdawson** [comment]
@nodejs/tsc would like at least one reviewer before landing.

**Review by codebytere — APPROVED** [review]

File: meetings/2020-09-24.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+* Anatoli Papirovski @apapirovski (TSC)
+* Beth Griggs @BethGriggs (TSC)
+* Сковорода Никита Андреевич @ChALkeR (TSC)
+* Colin Ihrig @cjihrig (TSC)
+* Mary Marchini @mmarchini (TSC)
+* Michael Dawson @mhdawson (TSC)
+* Myles Borins @MylesBorins (TSC)
+* Rich Trott @Trott (TSC)
+* Guy Bedford @guybedford (Guest)
+* Geoffrey Booth GeoffreyBooth (Guest)
```
Comment (codebytere) [inline meetings/2020-09-24.md:19]: should this have an `@`?

Comment (mhdawson) [inline meetings/2020-09-24.md:19]:
```suggestion
* Geoffrey Booth @GeoffreyBooth (Guest)
```

Comment (mhdawson) [reply meetings/2020-09-24.md]: fixed

**Review by BethGriggs — APPROVED** [review]

**Pull request merged**